exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,13 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function uppercaseNames(name) {
+    return name.toUpperCase();
+}
+
+console.log("HELLO " + uppercaseNames(mentor1));
```

Comment:
You should create a nested function (call another function within a function) to complete this exercise, instead of repeating "HELLO " each time you print out the shouty message.

Reply:
Thank You Doris, I've fixed it.